### ISSUE TITLE
Bug: Tabbing and focus trapping doesn't work properly when multiple modals are open

### ISSUE THREAD

**Issue by mike-vel** [ghosh/Micromodal#544]
When multiple modals are open, `Tab` or `Shift+Tab` causes the focus to land in unexpected places, making it impossible to navigate.

[Codepen demo](https://codepen.io/mike-vel/pen/YPyBQgo)

**Comment by dkniffin** [comment]
This should now be resolved in 0.7.0. Let me know if the issue continues.
